travelnet.lua

```diff
@@ -1,21 +1,60 @@
 
+local yellow, red, orange, blue, cyan, green, dark_green, violet, pink, magenta, brown, grey, dark_grey, black, white = ""
+
+if core.get_modpath("dye") then
+   yellow = "dye:yellow"
+   red = "dye:red"
+   orange = "dye:orange"
+   blue = "dye:blue"
+   cyan = "dye:cyan"
+   green = "dye:green"
+   dark_green = "dye:dark_green"
+   violet = "dye:violet"
+   pink = "dye:pink"
+   magenta = "dye:magenta"
+   brown = "dye:brown"
+   grey = "dye:grey"
+   dark_grey = "dye:dark_grey"
+   black = "dye:black"
+   white = "dye:white"
+end
+
+if core.get_modpath("mcl_dye") then
+   yellow = "mcl_dye:yellow"
+   red = "mcl_dye:red"
+   orange = "mcl_dye:orange"
+   blue = "mcl_dye:blue"
+   cyan = "mcl_dye:cyan"
+   green = "mcl_dye:green"
+   dark_green = "mcl_dye:dark_green"
+   violet = "mcl_dye:violet"
+   pink = "mcl_dye:pink"
+   magenta = "mcl_dye:magenta"
+   brown = "mcl_dye:brown"
+   grey = "mcl_dye:grey"
+   dark_grey = "mcl_dye:dark_grey"
+   black = "mcl_dye:black"
+   white = "mcl_dye:white"
+end
```

Comment:
it would make a lot more sense if this in a scoped table, so for example `dye_compat.red`, `dye_compat.yellow`, etc then making a ton of common name variables